src/runner/local_pool_runner.cpp

```diff
@@ -32,7 +28,7 @@ int doRunner(int argc, char* argv[])
     usleep(1000 * 500);
 
     for (const auto& m : req->messages()) {
-        faabric::Message result = sch.getFunctionResult(m.id(), 20000);
+        faabric::Message result = sch.getFunctionResult(m.id(), 20000 * 100);
```

Comment:
For the local pool runner it makes sense to have longer timeouts, as we may want to run some longer running functions for debug purposes.

As a reminder, the local pool runner is a `pool_runner` that can be invoked from the command line. For example, the following runs a LAMMPS simulation without requiring any changes to faasm (i.e. any new `lammps_runner` etc):

```bash
 WASM_VM=wamr inv run.pool lammps main --cmdline '-in faasm://lammps-data/in.controller.wall'
```